### ISSUE TITLE
[ACCESSIBILITY] - alt text on images cleanup

### ISSUE THREAD

**Issue by sptramer** [o3de/o3de.org#391]
In the migrated documentation, we have a large number of missing, inaccurate, misleading, or non-conventional styles of alt text for accessibility purposes. This is obviously a very big task to dump into a single ticket at the moment, but at minimum we should:

- Start identifying the major sections of the guide which need serious cleanup, and create additional tickets if needed
- Make cleanup efforts where sections are small or clearly going to stick around for a while
